socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
```

Comment:
Sure. Your call. 
I'm just try to share my guttural feedback. More verbose code is sometimes easier to read that code that raises little questions in your head. Like 
```python
print sum(int(num) for xs in l if xs > 50 for num in xs if int(num) < 100)
```
versus
```python
total = 0
for group in groups:
    for number in group:
        number = int(number)
        if number < 100:
            total += number
print total
```

I'm not trying to convince you of anything. Just some loud thinking.